hack/generator/pkg/codegen/pipeline/add_cross_resource_references.go

```diff
@@ -330,5 +330,18 @@ func newKnownReferencesMap(configuration *config.Configuration) map[referencePai
 			typeName: astmodel.MakeTypeName(configuration.MakeLocalPackageReference("microsoft.containerservice", "v1alpha1api20210501"), "ManagedClusterProperties"),
 			propName: "DiskEncryptionSetID",
 		}: true,
+		// PostgeSQL
+		{
+			typeName: astmodel.MakeTypeName(configuration.MakeLocalPackageReference("microsoft.dbforpostgresql", "v1alpha1api20210601"), "Network"),
+			propName: "PrivateDnsZoneArmResourceId",
+		}: true,
+		{
+			typeName: astmodel.MakeTypeName(configuration.MakeLocalPackageReference("microsoft.dbforpostgresql", "v1alpha1api20210601"), "Network"),
+			propName: "DelegatedSubnetResourceId",
+		}: true,
+		{
+			typeName: astmodel.MakeTypeName(configuration.MakeLocalPackageReference("microsoft.dbforpostgresql", "v1alpha1api20210601"), "ServerProperties"),
+			propName: "SourceServerResourceId",
+		}: true,
```

Comment:
Suggest pulling out the repeated calls to `MakeLocalPackageReference()` as locals to simplify:

```suggestion
// earlier
postgress20210601 := astmodel.MakeTypeName(configuration.MakeLocalPackageReference("microsoft.dbforpostgresql", "v1alpha1api20210601")


		{
			typeName: postgress20210601, "Network"),
			propName: "PrivateDnsZoneArmResourceId",
		}: true,
		{
			typeName: astmodel.MakeTypeName(postgress20210601, "Network"),
			propName: "DelegatedSubnetResourceId",
		}: true,
		{
			typeName: astmodel.MakeTypeName(postgress20210601, "ServerProperties"),
			propName: "SourceServerResourceId",
		}: true,
```

Reply:
Fixed (and for the other entries here so that they all follow the same pattern)